packages/devtools_app/lib/src/shared/connected_app.dart

```diff
@@ -36,62 +36,62 @@ class ConnectedApp {
   Future<bool> get isFlutterApp async => _isFlutterApp ??=
       await serviceManager.libraryUriAvailable(flutterLibraryUri);
 
-  bool get isFlutterAppNow {
+  bool? get isFlutterAppNow {
     assert(_isFlutterApp != null);
-    return _isFlutterApp;
+    return _isFlutterApp == true;
   }
 
-  bool _isFlutterApp;
+  bool? _isFlutterApp;
```

Comment:
can we make the private fields late instead of nullable?

Reply:
All of them can be validated before initialization in connectedAppInitialized. We do not want this getter to fail. So, leaving nullable.